apps/studio/src/features/editing-experience/data/databaseLayoutPreview.json

```diff
@@ -0,0 +1,65 @@
+{
+  "version": "0.1.0",
+  "layout": "database",
+  "page": {
+    "permalink": "/data/datasets/sample-dataset",
+    "title": "New database layout",
+    "description": "This is a layout where you can link your dataset from Data.gov.sg. Users can search through the table.",
+    "database": {
+      "title": "Link your database here",
+      "headers": ["Contribute type", "Amount"],
+      "items": [
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"],
+        ["Cell copy", "Cell copy"]
+      ]
+    }
```

Comment:
even though this feature is to allow agency users to add DGS component, for preview we can use native searchable table to 1) reduce additional DGS API call + 2) reduce latency (no need to wait for API)